custom_components/miele/binary_sensor.py

```diff
@@ -164,6 +164,7 @@ class MieleBinarySensorDefinition:
             data_tag="state|signalFailure",
             device_class=BinarySensorDeviceClass.PROBLEM,
             name="Failure",
+            icon="mdi:alert",
```

Comment:
My basic view on this is that we should adhere to HA recommendations if there is no special reason against. The device class PROBLEM has a default icon. Isn't that good enough?

Reply:
It's just to differentiate this sensor from the other binary called "info". This is related to a fault on the appliance, the other one is related to a generic message that pops up on the appliance. So, in this case I wanted to highlight the fault with the triangle, instead of the default rounded icon.
But, if you want, I can revert this.